EggSweeper/src/Animation.java

```diff
@@ -130,6 +142,17 @@ public void addHole(int xLoc, int yLoc, int sizeX, int sizeY) {
 			AniObject holeObject = new AniObject("hole", xLoc, yLoc, sizeX, sizeY, holeList);
 			holeObject.setVisible(true);
 			images.add(holeObject);
+			int i = images.size() - 1;
+			int j = 0;
+			Iterator<AniObject> holeBirdItr = images.iterator();
+			while (holeBirdItr.hasNext()) {
+				AniObject next = holeBirdItr.next();
+				if (next.toString() == "bird") {
```

Comment:
When comparing string, use the .equalsIgnoreCase() method

example: 
if(next.toString().equalsIgnoreCase("Bird")) 

Why:
== refers to the object reference to see if the two string refer to the **same string instance** 
equalsIgnoreCase() refers to the values of each string.